build/debian/tribler/debian/changelog

```diff
@@ -1,58 +1,3 @@
-tribler (7.0.0) unstable; urgency=medium
```

Comment:
This is history of older versions, I'd like to keep these in place.

Reply:
Agree. But this history only consists of versions until the `7.0.0` which makes `changelog` a bit suspicious.
Why other versions are missed?

I see two options here:
1. To not have versions at all (this option is implemented in the current PR)
2. To have all versions

For option number 2 we should somehow create a changelog from the scratch. I've tried a few ways of doing this (unsuccessfuly) and returned back to option number 1.

If you are able to create a full changelog, feel free to add a commit to the PR or add a comment to this conversation.

Reply:
Agree. I'll return the history.

Reply:
Done!